File: python/1035.py
valores = input()

[a1, b1, c1, d1] = valores.split()

a = int(a1)
b = int(b1)
c = int(c1)
d = int(d1)

if (b > c) and (d > a) and (c + d) > (a + b) and c and d > 0 and a % 2 == 0:
    print("Valores aceitos")
else:
    print("Valores nao aceitos")

File: c/1035.c
#include <stdio.h>

int main()
{
  int a, b, c, d;
  scanf("%d %d %d %d", &a, &b, &c, &d);

  int soma_ab = a + b;
  int soma_cd = c + d;

  if ((b > c && d > a) && (soma_cd > soma_ab) && (c > 0 && d > 0) && (a % 2 == 0))
  {
    printf("Valores aceitos\n");
  }
  else
  {
    printf("Valores nao aceitos\n");
  }
  return 0;
}